src/cqparts/codec/gltf.py

```diff
@@ -327,7 +327,8 @@ class GLTFExporter(Exporter):
 
     """
 
-    scale = 0.001  # mm to meters
+    # scale = 0.001  # mm to meters
+    scale = 1
```

Comment:
I like what was done with tolerance; making it configurable per export call.
So I'm doing the same for `scale`